metadata_service/proxy/atlas_proxy.py

```diff
@@ -706,6 +734,47 @@ def get_table_by_user_relation(self, *, user_email: str, relation_type: UserReso
                 cluster=res['cluster'],
                 schema=res['db'],
                 name=res['table']))
+        return results
+
+    def _get_resource_owned_by_user(self, user_id: str, resource_type: str) \
+            -> List[Union[PopularTable, DashboardSummary, Any]]:
+        """
+        ToDo (Verdan): Dashboard still needs to be implemented.
+        Helper function to get the resource, table, dashboard etc owned by a user.
+        :param user_id: User ID of a user
+        :param resource_type: Type of a resource that returns, could be table, dashboard etc.
+        :return: A list of PopularTable, DashboardSummary or any other resource.
+        """
+        owned_by_resources = list()
```

Comment:
the name seems weird to me, can it just be `results`  (like in `_get_resource_followed_by_user`) ? + can we instantiate empty lists with `[]` ?

Reply:
the word `results` is not pythonic in a way that it gives you no context. any other suggestion ?

Reply:
hm, the context should be really provided by function name, not the variable name itself. I'd settle for `resources`. btw. both this and _get_resource_followed_by_user function should have resource**s** (plural) in the name since we return a list.